packages/client/src/forms/index.ts

```diff
@@ -512,7 +510,6 @@ export interface IFormFieldBase {
   ignoreFieldLabelOnErrorMessage?: boolean
   ignoreBottomMargin?: boolean
   customQuestionMappingId?: string
-  ignoreMediaQuery?: boolean
```

Comment:
We could remove these properties from the Farajaland's form type as well. 
https://github.com/opencrvs/opencrvs-farajaland/blob/develop/src/form/types/types.ts#L588